tests/benchdnn/graph/custom_driver.cpp

```diff
@@ -69,8 +69,9 @@ int init_ref_memory_args(dnn_mem_map_t &ref_mem_map, dnn_mem_map_t &mem_map,
 
 int execute(const prb_t *prb, const args_t &args, res_t *res) {
     dnn_mem_t &dst = const_cast<dnn_mem_t &>(args.find(DNNL_ARG_DST));
-    size_t axis = prb->axis;
     auto ndims = dst.ndims();
+    size_t axis = prb->axis;
+    if (prb->axis < 0) { axis = prb->axis + ndims; }
```

Comment:
```suggestion
    const size_t axis = prb->axis < 0 ? (prb->axis + ndims) : prb->axis;
```

BTW, since it is passed by the user through benchdnn command line, do we need to verify `axis` in [0, ndims)?

Reply:
axis can be [-ndims, ndims), so I think negative values of axis is acceptable. And whether we need to verify the user input to be in the range of [-ndims, ndims), my understanding is that user should follow [our document](https://oneapi-src.github.io/oneDNN/dev_guide_op_genindex.html) to set the value of this attribute

Reply:
OK, let me add a check for axis :)
But I feel it's more useful to have such check at library side instead of benchdnn side. If we want to add a check at library side, we can only check at compilation stage (since ndims may not exist when get_partition).

Reply:
Thanks and yes, we need to have the similar check on library side at some places. But here i feel it's actually custom driver for reference.

Reply:
fixed, thanks!